EXAMPLES.md

```diff
@@ -0,0 +1,67 @@
+# PromQL usage examples for Fortigate Exporter
+
+PromQL is an advanced language and proper usage of it requires one to both understand
+the language itself and how the metrics one has are constructed.
```

Comment:
English is my second language so could be me, but that sounds kinda weird. 

WDYT about  `PromQL is an advanced language and proper usage of it requires first to understand the language itself and how the metrics one are constructed.`

Reply:
Yeah, it is a hard sentence to grok. What do you think of this new one?

Reply:
What are trying to bring across with organized? I'm not sure what you mean here

Reply:
I want to state that even if you are the world's best champion on PromQL, you still need to learn how each exporter works; as in what metrics are available to you, and how their labels associate between the metrics.

E.g. that we have a fortigate_version_info that can be used to get the build info (as opposed to adding the build label to all metrics, as another exporter might have had done).

Reply:
I see. :)